### DISCUSSION TITLE
False positive

### DISCUSSION THREAD

**Discussion by syuilo** [infinitered/nsfwjs#701] [Model Training]
nsfw.js predicts these images as porn. I used V3 model.

These pictures will not look very common, but they are all scenes from a common video (e.g. https://www.youtube.com/watch?v=kAFUYNVDl3k). Therefore, the problem arises when sampling some frames of a video or gif to determine nsfw.

![image](https://user-images.githubusercontent.com/4439005/174216287-01374ad2-3e4d-48d0-a63b-54abe7af7f8c.png)

![image](https://user-images.githubusercontent.com/4439005/174216300-5f91aee3-d123-4d25-97f3-ea30942e7146.png)

![88](https://user-images.githubusercontent.com/4439005/174218823-64cc59a2-272c-429c-804f-2851d50cd127.png)

![21](https://user-images.githubusercontent.com/4439005/174218550-835af67d-0fba-475d-9601-27175c7d935f.png)

![7](https://user-images.githubusercontent.com/4439005/174218592-f83b1d46-0811-47d6-bc7a-42ca8c64497d.png)

![171](https://user-images.githubusercontent.com/4439005/174218373-f3c7dc2d-cb40-4145-8d90-142a48c2f5b5.png)

![59](https://user-images.githubusercontent.com/4439005/174218484-c9587284-f9dc-4baf-863b-9aa1ef5e91e1.png)

![image](https://user-images.githubusercontent.com/4439005/174216412-170061f9-25dd-4acb-beb3-eda6ae7a57f7.png)

![image](https://user-images.githubusercontent.com/4439005/174216446-cf666fcb-31f7-4035-bbf0-ca6d8db24e30.png)

![image](https://user-images.githubusercontent.com/4439005/174218840-9e2a9f29-4be4-48ec-92a0-051e6f390aaf.png)

**Comment by GantMan** [comment]
That's so odd!  I will add these to training to help fix.
